core/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'java'
+}
+
+group 'org.mobilitydata'
+version "gtfs-validator-${System.getenv("versionTag")}"
+
+sourceCompatibility = 1.8
```

Comment:
Out of curiosity, we used to have 
```sourceCompatibility = JavaVersion.VERSION_11``` why was this change required?

Reply:
This makes sure that we can run Validator at Google.

Google's products are one step ahead but the technologies may be outdated :(

Reply:
Thanks!